data/test-cases/plugins/ms-teams-sync/attachments-sync/Not_allowed_attachment_size.md

```diff
@@ -0,0 +1,43 @@
+---
+# (Required) Ensure all values are filled up
+name: "Not allowed size of attachment on MM"
```

Comment:
Can we also write a scenario here for multiple attachments where there might be some that are allowed and others are not due to there size?

Reply:
I think that might be a case for Multiple attachment and will add this scenario in the file for multiple attachments.